department-of-veterans-affairs/next-build
# VACMS-0000 add tests for refactored queries

## Description
This PR builds on #110. Merge that one first! Relates to the refactors for emailContact, benefitsHub, wysiwyg, & richTextCharLimit1000. Adds tests for those new queries.

the intersection of NodeLandingPage and NodeOffice has some issues, i believe because the office pages referenced are unpublished. There is also a change to the field_meta_title -> field_meta_tags that isn't captured in our typing yet... but seemed a little out of scope for this work. See this old PR that nukes that field: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/9246/files

## Testing done
Used the api-explorer to generate mock data for the above, and created tests for these queries.

## Screenshots

```
yarn test
...
 PASS  src/data/queries/tests/benefitsHub.test.tsx
 › 1 snapshot written.
...
 PASS  src/data/queries/tests/emailContact.test.tsx
...
 PASS  src/data/queries/tests/richTextCharLimit1000.test.tsx
 › 1 snapshot written.
 PASS  src/data/queries/tests/wysiwyg.test.tsx
 › 1 snapshot written.
...
Snapshot Summary
 › 3 snapshots written from 3 test suites.

Test Suites: 3 skipped, 39 passed, 39 of 42 total
Tests:       11 skipped, 123 passed, 134 total
Snapshots:   3 written, 10 passed, 13 total
Time:        8.24 s
```
## QA steps

pull branch, run `yarn test` see everything pass. or see that this PR passes CI.

### Definition of Done
